bucket/openmpt.json

```
@@ -0,0 +1,37 @@
+{
+    "homepage": "https://openmpt.org/",
+    "description": "OpenMPT is a powerful audio application that makes writing music fun, easy and efficient.",
+    "license": "BSD-3-Clause",
+    "version": "1.28.05.00",
+    "architecture": {
+        "32bit": {
+            "url": "https://openmpt.org/download_file.php?file=OpenMPT-1.28.05.00-Setup.exe",
+            "hash": "07aaf66792dbe0d04f41cee842b639ab5959440f80e9683179aaafb86698bf6a"
+        },
+        "64bit": {
+            "url": "https://openmpt.org/download_file.php?file=OpenMPT-1.28.05.00-Setup-x64.exe",
+            "hash": "033c31cd9b20924bbb36cbe52a889c042b13448b8ca38302b5ca128417b2e0c2"
+        }
+    },
+    "innosetup": true,
+    "shortcuts": [
+        [
+            "mptrack,1.exe",
+            "OpenMPT"
+        ]
+    ],
+    "checkver": {
+        "url": "https://openmpt.org/download",
+        "regex": "OpenMPT ([\\d.]+)"
+    },
+    "autoupdate": {
+        "architecture": {
+            "32bit": {
+                "url": "https://openmpt.org/download_file.php?file=OpenMPT-$version-Setup.exe"
+            },
+            "64bit": {
+                "url": "https://openmpt.org/download_file.php?file=OpenMPT-$version-Setup-x64.exe"
+            }
+        }
```

Something like this should work. Needs testing

```suggestion
        },
        "hash": {
            "url": "https://openmpt.org/download",
            "regex": "$basename.*?$sha1"
        }
```

---

digest file will be better

---

Just use `$url.digests` with regex `$sha1`